src/middelwares/validateNumbers.ts

```diff
@@ -0,0 +1,19 @@
+import express, { Request, Response, NextFunction } from "express";
+const app = express();
+
+app.use(express.json());
+
```

Comment:
You should not create an instance of the express server in every file.
The entire application should use the same instance which is defined in the entry point file.